containers/groovy-examples/docker-compose.yml

```diff
@@ -2,9 +2,7 @@ version: "3.4"
 
 services:
   deephaven:
-    image: ghcr.io/deephaven/server-netty:latest
-    labels:
-      - 'traefik.http.routers.deephaven.rule=Host(`localhost`)'
+    image: ghcr.io/deephaven/server-slim-jetty:latest
```

Comment:
There is no `-jetty` suffix - `ghcr.io/deephaven/server-slim` for 0.17.0 (and latest) will now be jetty.